sysl2/sysl/seqs/visitor.go

```diff
@@ -179,17 +197,23 @@ func (e *StatementElement) isLastStmt(i int) bool {
 type SequenceDiagramVisitor struct {
 	AppLabeler
 	EndpointLabeler
-	w       *SequenceDiagramWriter
-	m       *sysl.Module
-	visited map[string]int
-	symbols map[string]*_var
+	w          *SequenceDiagramWriter
+	m          *sysl.Module
+	visited    map[string]int
+	symbols    map[string]*_var
+	stdout     io.Writer
+	stderr     io.Writer
```

Comment:
I don't see a valid use for `stdout` and `stderr`. Remove unless needed.

Reply:
It's needed for log validation as part of test case.